src/vue-components/setting.vue

```diff
@@ -33,10 +33,11 @@
     computed: {
       selected: {
         get: function () {
+          let rv
           if (typeof this.data.currentTextValue === 'function') {
-            return this.data.currentTextValue()
+              rv = this.data.currentTextValue()
           }
-          return null
+          return rv
```

Comment:
I'm wondering if we can simplify the above to:
`return (typeof this.data.currentTextValue === 'function')? this.data.currentTextValue() : null`

Reply:
yes that's a good suggestion. this was leftover from some of my earlier failed debugging.